jcl/src/java.base/share/classes/java/lang/StackWalker.java

```diff
@@ -306,6 +309,29 @@ public void forEach(Consumer<? super StackFrame> action) {
 		 * @return StackTraceElement
 		 */
 		StackTraceElement toStackTraceElement();
+
+		/*[IF Java10]*/
+		/**
+		 * This method must be implemented in the implementing class.
+		 * @throws UnsupportedOperationException if this method is not overridden
+		 * @return MethodType containing the parameter and return types for the associated method.
+		 * @since 10
+		 */
+		default MethodType getMethodType() {
+			throw new UnsupportedOperationException();
+		}
+
+		/**
+		 * This method must be implemented in the implementing class.
```

Comment:
This is kind of throw away text unless it clarifies the expected implementation / behaviour of the method.

Reply:
Removed.